src/utils.js

```diff
@@ -3,31 +3,39 @@ import type { VLQMap, SourceMapStringifyOptions } from './types';
 
 import path from 'path';
 
+// For some reason path.isAbsolute barely works... Regex to the rescue?
```

Comment:
On Unix, `path === path.posix` and on Windows `path = path.win32`. So `path.isAbsolute` is dependent on the host OS at runtime.

Reply:
Yeah... which kinda sucks for this use-case